presentation-api/controlling-ua/PresentationConnection_onmessage-manual.html

```diff
@@ -38,6 +38,7 @@
 
     presentBtn.onclick = () => {
         presentBtn.disabled = true;
+        const stash = new Stash('9bf08fea-a71a-42f9-b3c4-fa19499e4d12', 'f1fdfd10-b606-4748-a644-0a8e9df3bdd6');
```

Comment:
Okay. I'll move declarations of two IDs to `common.js` and give them names which help understand their usage (`toController` and `toReceiver`).